app/views/works/main.html.erb

```diff
@@ -0,0 +1,65 @@
+<main>
+  <section class="spotlight">
+    <h2 class="spotlight__header">
+      <span class="spotlight__header--prefix">Media Spotlight:</span>
+      <%= link_to @top_work.title, work_path(@top_work) %>
+    </h2>
+    <p class="spotlight__description">
+      <%= @top_work.votes.count %> votes - <%= @top_work.description %>
+    </p>
+
+  </section>
+
+<hr class="root__hr">
+
+<section class="top-ten__container">
+  <section class="top-ten__list-container">
+  <h2 class="top-ten__header">Top Movies</h2>
+  <ul class="list-group top-ten__list">
+    <% @movies.first(10).each do |work| %>
```

Comment:
Notice how these tables are similar.  You could DRY up your code with partials to generate these tables.